src/kaleidoscope/plugin/FocusSerial.h

```diff
@@ -75,6 +75,9 @@ class FocusSerial : public kaleidoscope::Plugin {
   void read(uint16_t &u16) {
     u16 = KeyboardHardware.serialPort().parseInt();
   }
+  void read(typename HARDWARE_IMPLEMENTATION::LEDDriverProps::LEDCountType &u) {
```

Comment:
use the macro for led count type?

Reply:
That was the first thing I went with, but we can't do that, compiler says "no thanks". I'll try again and paste the error later tonight.